_migrations/is-migration-assistant-right-for-you.md

```diff
@@ -1,244 +1,63 @@
 ---
 layout: default
-title: Is migration assistant right for you?
+title: Is Migration Assistant right for you?
 nav_order: 5
 ---
 
-Before deciding if this is the right tool for you, it's important to assess your specific needs and understand the available tools for performing an upgrade or migration.
+# Is Migration Assistant right for you?
+
+Before deciding if Migration Assistant is right for you, it's important to assess your specific needs and understand the available tools for performing an upgrade or migration.
 
 Migration Assistant addresses gaps found in other migration solutions, but in some cases, alternative tools may be a better fit.
 
 For instance, if you need to upgrade more than one major version—such as moving from Elasticsearch 6.8 to OpenSearch 2.3—Migration Assistant allows you to do this in a single hop. In contrast, other options like rolling upgrades or snapshot restore would require multiple steps, as they cannot handle major version jumps without reindexing your data. Additionally, if you need to capture live traffic and perform a zero-downtime migration, Migration Assistant would be the right choice.
```

Comment:
minor -- I think it should be an em-dash between "OpenSearch 2.3" and "Migration Assistant" (and we should probably make it OpenSearch 2.15 for good measure)